pages/api/performance/create.js

```diff
@@ -0,0 +1,65 @@
+import prisma from '@prisma/index'; // Prisma client
+import { getSession } from 'next-auth/client'; // Session handling
+
+export default async (req, res) => {
+  // Collect session from request
+  const session = await getSession({ req });
+
+  // If user is authenticated and is an admin
+  if (session && session.isAdmin) {
+    // Collect performance information
+    const {
+      name,
+      academic_level,
+      performers,
+      choreographers,
+      competition_level,
+      dance_size,
+      dance_entry,
+      dance_style,
+      dance_title,
+      event_id,
+      school_id,
+    } = req.body;
+
+    // If all exist
+    if (
```

Comment:
Some fo these fields are optional in the database schema. Do we want to enforce all in the POST request?

Reply:
I think it would make sense to make them required in the schema unless there are certain fields we won't know when we create the performance @thepenguinco

Reply:
Will make the schema match the validation.